packages/markdown-docx/test/DocxTransformer.test.js

```diff
@@ -17,12 +17,12 @@
 'use strict';
 
 const fs = require('fs');
-const DocxTransformer = require('./DocxTransformer');
+const DocxTransformer = require('../src/DocxTransformer');
 
 let docxTransformer = null;
 
 // @ts-ignore
-beforeAll(() => {
+beforeEach(() => {
```

Comment:
Does this change matter? We are anyway skipping the test.

Reply:
@algomaster99 
Gave me errors if it made `beforeAll`.

Reply:
What was the error?

Reply:
@algomaster99 
`ReferenceError: beforeAll is not defined`

Reply:
Okay. But did you understand why that happened?

Reply:
Yes. Mocha has no beforeAll function as per my understanding. Hence it cannot use that.